collect_app/src/test/java/org/odk/collect/android/widgets/SelectWidgetTest.java

```diff
@@ -63,17 +63,21 @@ public void whenQuestionHasAudio_audioButtonUsesIndexAsClipID() throws Exception
                 new SelectChoice("2", "2")
         );
 
+        for (int i = 0; i < selectChoices.size(); i++) {
+            selectChoices.get(i).setIndex(i);
+        }
+
         when(formEntryPrompt.getSpecialFormSelectChoiceText(selectChoices.get(0), FormEntryCaption.TEXT_FORM_AUDIO)).thenReturn("file://blah1.mp3");
         when(formEntryPrompt.getSpecialFormSelectChoiceText(selectChoices.get(1), FormEntryCaption.TEXT_FORM_AUDIO)).thenReturn("file://blah2.mp3");
 
-        setupMockReference("file://blah1.mp3", referenceManager);
-        setupMockReference("file://blah2.mp3", referenceManager);
+        setupMockReference("file://blah1.mp3", "ref1", referenceManager);
```

Comment:
Yeah I prefer just having a String like this. I think using a "real" looking URI shows too much knowledge in the test of what the `ReferenceManager` does - we don't strictly care about how it works in these tests.

Reply:
Actually I think what would be nicer after looking at it again would be to return a `String` from `setupMockReference` so that the test could just pass that around instead of us having to trace the value through ourselves when we're reading/writing the test